projects/10/eva_submitter/README.md

```diff
@@ -0,0 +1,12 @@
+# EVA Submitter: From a BrAPI compliant Database to EVA submission format
+
+## BrAPI to EVA mapping
+
+A mapping between BrAPI and EVA has been made [here](../EVA_Submission_template_V1_1_3_mapping_to_BRAPI_v2.xlsx)
```

Comment:
This is currently project 10: https://github.com/elixir-europe/BioHackathon-projects-2019/tree/master/projects/10
The link you have here (projects/10/eva_submitter/README.md) would go to a file that currently does not exist (../EVA_Submission_template_V1_1_3_mapping_to_BRAPI_v2.xlsx) and that is not listed as part of this PR. Could you please double check all files required are included in the PR?